code/modules/power/tesla/coil.dm

```diff
@@ -95,5 +115,15 @@
 
 	return ..()
 
+/obj/machinery/power/grounding_rod/attack_hand(mob/user)
+	if(can_buckle && isliving(user.pulling) && user.a_intent == INTENT_GRAB && !has_buckled_mobs())
```

Comment:
It's because I need to check for this before forcemoving the mob. And I can't call `user_buckle_mob()` before forcemoving because it checks loc.